src/style.css

```diff
@@ -1 +1,66 @@
 /* styles go here */
+.form_field {
+  display: block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.fieldset-1 {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.fieldset-2 {
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+.fieldset-3 {
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+.fieldset-4 {
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+#F1 {
```

Comment:
Avoid using IDs in CSS. IDs are not reusable and can lead to specificity issues. Consider using classes instead.

Reply:
Not fixed. Use classes instead to avoid repeating the same styles for each id